src/plugin/file/kml/kmlnodelayerui.js

```diff
@@ -1,11 +1,13 @@
 goog.provide('plugin.file.kml.KMLNodeLayerUICtrl');
+goog.provide('plugin.file.kml.KMLNodeLayerUICtrl.UIEventType');
```

Comment:
Adding in these kinds of missing provide/requires can be risky and result in circular dependencies. The original addition of this event type enum to this class was incorrect. These sorts of event types should really be added to utility classes.

Reply:
I think I'll leave this for now